bif_arg.cc

```diff
@@ -76,6 +75,15 @@ void BuiltinFuncArg::PrintCDef(FILE* fp, int n, bool runtime_type_check) {
     fprintf(fp, builtin_func_arg_type[type].accessor, buf);
 
     fprintf(fp, ");\n");
+
+    // For pointer types, validate that the return value is valid to avoid accessing null pointers.
+    std::string_view c_type{builtin_func_arg_type[type].c_type};
+    if ( c_type.back() == '*' ) {
+        fprintf(fp, "\t if ( %s == nullptr ) {\n", name);
```

Comment:
Would that be better than before line 60, where we already would access a nullptr - runtime_type_check is only true for variable args, so maybe the combination doesn't really happen right now.

Reply:
There are definitely places where the second one can get generated without the first one. I added a null check on the first one plus a flag for whether it got generated to avoid generating it twice.